examples/append_conc_to_h5file.py

```diff
@@ -0,0 +1,21 @@
+"""Edit the concentration parameter of a saved HDF5 file
+========================================================
+
+Loads in a previously saved HDF5 dataset. If the attribute
+'concentration' does not exist in the acq_params, this will 
+add the concentration defined by 'actual_conc' into acq_params
+under 'concentration'. If the attribute already exists it will
+simply replace the current concentration with the value of 
+'actual_conc'.
+"""
+from pyspecdata import *
+import h5py,os
+data_dict = {"filename":'220126_Ras_M67R1a_capProbe.h5', #file that is being edited
+        "file_location":"ODNP_NMR_comp/ODNP", #location of file
+        "actual_conc": 72e-6} #concentration of the dataset in M
```

Comment:
Logically, the concentration is a separate thing... This is not information about where the file is located as is true of the dictionary... It is the concentration information you are incorporating into that file.

Reply:
so I changed the name to data_info which I think encapsulates file name, location and the concentration but if you think this is still weird I can adjust easily